docs/data/sdks/sdk-maintenance-and-support.md

```diff
@@ -0,0 +1,93 @@
+| :material-android: [Android SDK](./android-kotlin/index.md)<br/>:octicons-package-16: [`com.amplitude:analytics-android`](https://mvnrepository.com/artifact/com.amplitude/analytics-android)<br/>:material-github: [Amplitude-Kotlin](https://github.com/amplitude/Amplitude-Kotlin)                                                                                                     | 1.x       | GA             | 2022-06-28 |                                             |
+| :material-android: [Android SDK](./android/index.md)<br/>:octicons-package-16: [`com.amplitude:android-sdk`](https://mvnrepository.com/artifact/com.amplitude/android-sdk)<br/>:material-github: [Amplitude-Android](https://github.com/amplitude/Amplitude-Android)                                                                                                                      | 3.x       | End-of-support | 2021-12-16 | EOL: 2021-12-16<br/>Bad release (3.35.1)    |
+| :material-android: [Android SDK](./android/index.md)<br/>:octicons-package-16: [`com.amplitude:android-sdk`](https://mvnrepository.com/artifact/com.amplitude/android-sdk)<br/>:material-github: [Amplitude-Android](https://github.com/amplitude/Amplitude-Android)                                                                                                                      | 1.x -2.x  | Maintenance    | 2014-05-01 | MD: 2023-06-14                              |
+| :material-language-typescript: [Browser SDK](./browser-2/index.md) <br/> :octicons-package-16: [`@amplitude/analytics-browser`](https://www.npmjs.com/package/@amplitude/analytics-browser)<br/>:material-github: [Amplitude-TypeScript](https://github.com/amplitude/Amplitude-TypeScript)                                                                                               | 2.x       | GA             | 2023-06-14 |                                             |
+| :material-language-typescript: [Browser SDK](./typescript-browser/index.md) <br/> :octicons-package-16: [`@amplitude/analytics-browser`](https://www.npmjs.com/package/@amplitude/analytics-browser)<br/>:material-github: [Amplitude-TypeScript](https://github.com/amplitude/Amplitude-TypeScript)                                                                                      | 1.x       | Maintenance    | 2022-06-29 | MD: 2023-06-29                              |
+| :material-language-typescript: [Browser SDK](./marketing-analytics-browser/index.md) <br/> :octicons-package-16: [`@amplitude/marketing-analytics-browser`](https://www.npmjs.com/package/@amplitude/arketing-analytics-browser)<br/>:material-github: [Amplitude-TypeScript](https://github.com/amplitude/Amplitude-TypeScript)                                                          | 1.x       | Maintenance    | 2022-06-29 | MD: 2023-06-29<br/>Replaced by Browser 2.0  |
```

Comment:
these two should still be in maintenance announcement (phase 2) rather than maintenance (phase 3) right?

Reply:
Do we need to mention Marketing Analytics here specifically?

Reply:
> these two should still be in maintenance announcement (phase 2) rather than maintenance (phase 3) right?

I was hoping to spare us the announcement since we are just creating this doc - so current SDKs are "grandfathered".

> Do we need to mention Marketing Analytics here specifically?

I think so. I want to call out that it is no longer a recommended SDK.